modules/openshift-developer-cli-sample-applications.adoc

```diff
@@ -0,0 +1,114 @@
+[id='odo-sample-applications']
+= Sample applications
+include::openshift-developer-cli-attributes.adoc[]
+
+:context: odo-sample-applications
+
+{odo-title} is compatible with any language or run time listed within the OpenShift catalog of component types. In order to access the component over the web, you can create a URL using `odo url create`.
+
+Example:
+
+[source,sh]
```

Comment:
If this is output from the "odo url create" command, we don't use any source language. So remove the "source,sh" from the code block.

Also, you could also put the "$ odo url create" command at the top of the code block so it's more obvious that it's output.